src/docs/product/alerts-notifications/index.mdx

```diff
@@ -8,19 +8,40 @@ description: "Sentry categorizes notifications into alerts and other types of no
 
 Sentry categorizes notifications broadly into alerts and non-alert notifications.
 
-## Alerts
+## Error Alerting
 
-[Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
+Alerts notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
 
-- **Errors**: Sentry automatically groups individual error events into Issues based on stack trace and other factors. You can then set alerts on changes in those issues (such as new issues or issues affecting many users), or on the raw events (such as frequency of events with a given tag).
+### Issue Alerts
 
-- **Performance**: You can alert on performance metrics like latency, apdex, failure rate, and throughput.
+Sentry automatically groups individual error events into issues based on stack trace and other factors. You can then set alerts on changes in those issues (such as new issues or issues affecting many users), or on the raw events (such as frequency of events with a given tag).
```

Comment:
Do we need to explain grouping here? It seems like an implementation detail that's less important than our topic, which is alerting.

"you can then set alerts..." this makes it seem like it's a second step, but I don't think it is. Perhaps more clear as "Issue alerts are triggered when you need them, such as when a new issue arises or affects a threshold of users or frequency."